cli/src/commands/sidekick.rs

```diff
@@ -0,0 +1,208 @@
+use anyhow::{bail, Context, Result};
+use clap::Parser;
+use cli_core::{
+	ctx,
+	rivet_api::{self},
+};
+use console::Term;
+use serde::Serialize;
+use serde_json::{json, Value};
+
+use crate::util::{
+	global_config,
+	struct_fmt::{self, Format},
+};
+
+#[derive(Parser)]
+pub enum SubCommand {
+	/// Get the link for the user to sign in
+	GetLink,
+	/// Long poll the server to check if the user has signed in
+	WaitForLogin {
+		/// The token to poll for
+		#[structopt(short, long)]
```

Comment:
Do we need `short` for sidekick commands?

`structopt` macro should be `clap` to be consistent with the rest of the repo, didn't realize this even worked.

Reply:
I think I had this generated, but didn't think about the context of how `short`
is used, so this is good. I'll change it out